dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -17,15 +17,30 @@
 #include "utils/datetime.h"
 #include "utils/fileutils.h"
 #include "utils/signal-monitor.h"
+#include "utils/stringutils.h"
+
+static struct rumcontext {
+    char application_id[200];
+    char session_id[200];
+    char view_id[200];
+} rumcontext;
+
+static struct maincontext {
```

Comment:
Isn't it the opposite? `struct context` is the type and the variable is named `maincontext`?

Reply:
Yes...again forgot to change those :(